src/website/views/templates/app-layout.pug

```diff
@@ -0,0 +1,934 @@
+                  a.chi-dropdown__menu-item(href="#")
+                    | Deutsch
+                  a.chi-dropdown__menu-item(href="#")
+                    | 简体中文
+                  a.chi-dropdown__menu-item(href="#")
+                    | 日本語
+              .chi-footer__links
+                ul
+                  li
+                    a(href='https://www.lumen.com/en-us/about.html') About Us
+                  li
+                    a(href='https://www.centurylink.com/aboutus/community/community-development/programs-for-customers-with-disabilities.html') Accessibility
+                  li
+                    a(href='https://jobs.lumen.com', target='_blank') Careers
+                  li
+                    a(href='https://www.lumen.com/en-us/contact-us.html') Contact Us
+                  // OneTrust Cookies Settings button start
+                  li
+                    a.optanon-toggle-display(href='#') Cookie Settings
+                  // OneTrust Cookies Settings button end
+                  li
+                    a(href='https://www.centurylink.com/aboutus/legal.html', target='_blank') Legal
+                  li
+                    a(href='https://www.lumen.com/en-us/about/legal/acceptable-use-policy.html', target='_blank') Legal Notices
+                  li
+                    a(href='https://www.lumen.com/en-us/about/legal/privacy-notice.html') Privacy
+                  li
+                    a(href='https://www.centurylink.com/aboutus/company-information/public-policy.html', target='_blank') Public Policy
+                  li
+                    a(href='https://www.lumen.com/en-us/about/legal/commission-for-complaints-for-telecom-television-services-ccts', target='_blank') Support
+                  li
+                    a(href='https://www.lumen.com/help/en-us/home.html') Canadian Ombudsman
+                  li
+                    a(href='https://assets.centurylink.com/is/content/centurylink/UK-Gender-Pay-Gap-Report-2018?Creativeid=621ee105-c759-4597-b947-de420d1e731c', target='_blank') Gender Pay Gap 2018 (UK)
+                  li
+                    a(href='https://www.centurylink.com/aboutus/legal/latam/tariffs-and-regulatory-information.html') Legal (Latin America)
+                  li
+                    a(href='https://www.lumen.com/en-us/about/legal/de-legal-notices.html') Legal Notices (Germany)
+                  li
+                    a(href='https://assets.centurylink.com/is/content/centurylink/Modern-Slavery-Statement-2019?Creativeid=fe9a3e89-5681-4142-95fa-5785b3cef63f', target='_blank') Modern Slavery Statement 2019 (UK)
+                  li
+                    a(href='https://www.lumen.com/en-us/about/legal/uk-tax-strategy.html') UK Tax Strategy
+                .chi-footer__copyright
+                  | &copy; 2021 Lumen Technologies. All Rights Reserved. Lumen is a registered trademark in the United States, EU and certain other countries.
+
+    :code(lang="html")
```

Comment:
As I recall, we agreed we would implement a configurable web component with predefined slots. Should the `Web Component` tab be visible here? Or we will add it once the web component is available?

Reply:
@janibolkvadze we would add `Web Component` tab when the web component is ready. Expectd before the release of the template ;)